interfaces/coin-sig.repl

```diff
@@ -0,0 +1,54 @@
+; --------------------------------------------------------------------------
+; --------------------------------------------------------------------------
+; Using formal verification in Pact interfaces to create example coin.
+
+(begin-tx)
+
+(define-keyset "test-keyset" (sig-keyset))
+
+(module acct-module "test-keyset"
+  @doc "account schema and table module"
+
+  (defschema account
```

Comment:
I'd like to figure out why `@model [(invariant (>= balance 0))]` didn't work for you and add it here.